trlx/trainer/accelerate_base_trainer.py

```diff
@@ -427,10 +427,19 @@ def evaluate(self):  # noqa: C901
                 # in online setting, compute the reward for validation
                 if self.reward_fn:
                     logger.info("Computing rewards")
-                    rewards = torch.tensor(
-                        self.reward_fn(samples=str_samples, prompts=str_prompts, outputs=str_outputs, **metadata),
-                        dtype=float,
+                    rewards = self.reward_fn(
+                        samples=str_samples,
+                        prompts=str_prompts,
+                        outputs=str_outputs,
+                        model_tok=self.tokenizer,
```

Comment:
Missed a spot. Should have been renamed from `model_tok` to `tokenizer`.

![image](https://github.com/CarperAI/trlx/assets/1885333/f7201248-13d0-4056-a744-26871ea8a4b1)

Reply:
Thanks for reporting this!